Sources/Aperture/Aperture.swift

```diff
@@ -40,7 +40,8 @@ public final class Aperture: NSObject {
     highlightClicks: Bool,
     screenId: CGDirectDisplayID = .main,
     audioDevice: AVCaptureDevice? = .default(for: .audio),
-    videoCodec: String? = nil
+    videoCodec: String? = nil,
+    isFlippedCropRect: Bool? = false
```

Comment:
This parameter needs to be documented in a doc comment. It should say what it does and what it's useful for.

Reply:
And it's should not be an optional.